api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -123,8 +128,16 @@ public void checkFreeTierBillingUsage() {
     sendAlertsForCostThresholds(usersToThresholdCheck, previousUserCosts, userCosts);
   }
 
+  private int compareCosts(final double a, final double b) {
```

Comment:
I wrote a little utility class for this so you don't have to remember the convention. org.pmiops.workbench.utils.Comparables

If you wanted to use it here, you'd need to have a little Comparable class wrapping the fuzzyCompare method. Assuming the convention is the same. I think you'd have make a comparable FuzzyCost class to wrap Double or something though, so not a huge win.